template_content/{% if use_github_actions %}.github{% endif %}/workflows/checks.yaml.jinja

```diff
@@ -89,6 +89,9 @@ jobs:
         run: npm run test
 {%- endif %}
 
+      - name: Scan TEAL files for issues
+        run: algokit task analyze . # add --diff flag if you want output stability checks instead
```

Comment:
Same here, the above test run will result in template substituted TEAL files in the .algokit directory and additionally a user should have commit to source control anyways, as .algokit is not git ignored.